src/classes/GE_PaymentServices.cls

```diff
@@ -195,6 +196,13 @@ public with sharing class GE_PaymentServices {
         public String campaignCode;
     }
 
+    public with sharing class AchData {
+        public String achCode;
+        public String consent;
+        public String type;
```

Comment:
```suggestion
        public String holderType;
```

Reply:
@vmartinov-sforg We can't really change this field since it is required to be `type` for the ACH purchase call request.